.github/Issue_Template/ADD_issue.yml

```diff
@@ -0,0 +1,54 @@
+name: 🚀 New Issue
+description: Use this to create a new issue
+title: "[Add] <write what you want to add>"
+labels: ["new issue", "good first issue"]
+body:
+  - type: markdown
+    attributes:
+      value: "Thanks for submitting an issue! Before you proceed with the submission, please provide the following information."
+  - type: textarea
+    id: description-feature
+    attributes:
+      label: "Details"
+      description: Please describe the details in a clear and detailed manner.
+      placeholder: Please enter your detailed description of the new issue here. 
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots-feature
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: checkboxes
+    id: contribution-terms
+    attributes:
+      label: "Type of Contribution"
+      options:
+        - label: "Update to an existing Animated Button"
```

Comment:
Which button?

Reply:
My bad @roshan1337d was working on same issue template for other repo forgot to change it.